app/views/sessions/index.html.erb

```diff
@@ -6,22 +6,81 @@
+        <%= link_to 'All dates', sessions_path, class: "flex min-w-fit items-center justify-center px-4 py-2 rounded-full font-medium #{active_pill_class}" %>
         <% current_conference.session_dates.each do |session_date| %>
           <%= link_to session_date.strftime("%a, %d"),
                       sessions_path(on_date: session_date),
-                      class: "flex min-w-fit items-center justify-center px-4 py-2 font-medium rounded-full #{session_pill_class(session_date)}" %>
+                      class: "flex min-w-fit items-center justify-center px-4 py-2 font-medium rounded-full #{active_pill_class(session_date)}" %>
         <% end %>
       </div>
 
-      <div class="flex items-center justify-center ml-3 p-2 bg-white rounded-full min-w-[40px] min-h-[40px]">
-        <%= inline_svg_tag("icons/filters.svg", size: "18") %>
+      <div class="flex items-center justify-center ml-3 p-2 rounded-full min-w-[40px] min-h-[40px] <%= active_filters? ? "bg-red text-white" : "bg-white text-red" %>">
+        <div class="relative" data-controller="dropdown">
+          <button type="button" data-action="dropdown#toggle click@window->dropdown#hide" class="flex items-center justify-center p-2 rounded-full">
+            <%= inline_svg_tag("icons/filters.svg", size: "18") %>
+          </button>
+
+          <div
+            data-dropdown-target="menu"
+            class="hidden transition transform origin-top-right absolute right-0"
+            data-transition-enter-from="opacity-0 scale-95"
+            data-transition-enter-to="opacity-100 scale-100"
+            data-transition-leave-from="opacity-100 scale-100"
+            data-transition-leave-to="opacity-0 scale-95">
+            <div class="pt-2 pb-2 bg-white rounded-lg shadow-lg flex-col inline-flex absolute top-2 right-0 w-40 max-w-40">
+              <%= form_with url: sessions_path, class: "grid", method: :get do |form| %>
+                <%= form.hidden_field :on_date, value: params[:on_date] %>
+
+                <% SessionQuery::STATUS_SCOPES.each do |name| %>
+                  <div class="self-stretch px-4 py-2 items-center gap-2 inline-flex">
+                    <%= form.check_box name, checked: params[name] == "1", class: "w-4 h-4 rounded text-red", skip_base_styling: true %>
+                    <div class="text-gray-800 font-medium">
+                      <%= form.label name do %>
+                        <%= t("views.status_filters.#{name}") %>
+                      <% end %>
+                    </div>
+                  </div>
+                <% end %>
+                <div class="h-6 px-2.5 py-4 rounded justify-center items-center gap-2.5 inline-flex">
+                  <div class="text-lg text-red italic underline font-black">
+                    <%= form.submit t("views.status_filters.submit"), name: "", class: "text-lg text-red italic underline font-black" %>
+                  </div>
```

Comment:
We need to update the dropdown shadow, items font weight and remove the button in order to match the designs;
<img width="1431" alt="Screenshot 2024-08-13 at 6 17 12 p m" src="https://github.com/user-attachments/assets/91db9364-6b75-4c43-bba5-66e4dea5cc85">
<img width="1431" alt="Screenshot 2024-08-13 at 6 17 06 p m" src="https://github.com/user-attachments/assets/33fbda2f-e563-4f05-a0df-054daa9d5a33">

Reply:
We also need to position the dropdown a bit more to the right and bottom to match the designs.

In order to submit the filters after a checkbox change you could use the "form" stimulus controller.

Also, not a blocker (nice to have) the filters button is not filling with the red background when the dropdown is opened